pkg/executor/push_test.go

```diff
@@ -252,74 +252,99 @@ func TestImageNameTagDigestFile(t *testing.T) {
 	testutil.CheckErrorAndDeepEqual(t, false, err, want, got)
 }
 
-var calledExecCommand = []bool{}
-var calledCheckPushPermission = false
+var checkPushPermsCallCount = 0
 
-func setCalledFalse() {
-	calledExecCommand = []bool{}
-	calledCheckPushPermission = false
+func resetCalledCount() {
+	checkPushPermsCallCount = 0
 }
 
 func fakeCheckPushPermission(ref name.Reference, kc authn.Keychain, t http.RoundTripper) error {
-	calledCheckPushPermission = true
+	checkPushPermsCallCount++
 	return nil
 }
 
 func TestCheckPushPermissions(t *testing.T) {
 	tests := []struct {
-		description           string
-		Destination           []string
-		ShouldCallExecCommand []bool
-		ExistingConfig        bool
+		description                     string
+		CacheRepo                       string
```

Comment:
nit (not blocking): All of these fields should be unexported.

Reply:
Updated in c55854f7